packages/one-app-bundler/__tests__/webpack/ssr-css-loader/__snapshots__/index.spec.js.snap

```diff
@@ -10,24 +10,25 @@ exports[`SSR CSS loader should remove css-loader's css-base, replace its push an
 
 exports[`SSR CSS loader should remove css-loader's css-base, replace its push and export the locals 5`] = `
 "
-      // some other line
+      // Imports
       
-      // imports
-
-
-      // module
-      require(\\"/path/to/one-app-bundler/webpack/ssr-css-loader/css-base.js\\")().push([module.id, \\"/**comment**/.react-select__One___hIKe {
-  position: relative; }
-
-.react-select__also___8ObpT {
-  background-color: #f9f9f9; }
-\\", \\"\\"]);
-
-      // exports
+      // Module
+      require(\\"/path/to/one-app-bundler/webpack/ssr-css-loader/css-base.js\\")().push([module.i, \\".my-root__ErrorLayout__ErrorLayout___2B91F {
+  margin: 0 auto;
+}
+
+.my-root__ErrorLayout__main___2IftH {
+  max-width: 1000px;
+  width: 100%;
+  margin: 0 auto;
+  padding: 10px;
+}\\", \\"\\"]);
+      // Exports
       exports = module.exports = {
-      	\\"One\\": \\"react-select__One___hIKe\\",
-      	\\"also\\": \\"react-select__also___8ObpT\\",
+      	\\"ErrorLayout\\": \\"my-root__ErrorLayout__ErrorLayout___2B91F\\",
+      	\\"main\\": \\"my-root__ErrorLayout__main___2IftH\\"
       };
+      module.exports = exports;
```

Comment:
The source replaces `exports.locals =` with `exports = module.exports =`. Would this last line be needed?

Could `module.export = exports;` be safely removed? Maybe before this:

https://github.com/americanexpress/one-app-cli/blob/bdb322109ecb01a6cb4b0b7936ff87eefb97d44b/packages/one-app-bundler/webpack/loaders/ssr-css-loader/index.js#L37-L40

Reply:
I tried removing this and the test cases failed. This replacement is definitely being used from what the snapshot diffs tell me.